src/dos/dos.cpp

```diff
@@ -1255,6 +1274,9 @@ class DOS:public Module_base{
 		dos.version.minor=0;
 		dos.direct_output=false;
 		dos.internal_output=false;
+
+		const Section_prop* section = static_cast<Section_prop*>(configuration);
+		set_ver((char *)section->Get_string("ver"), true);
```

Comment:
C++ static_cast.

Reply:
+1 for using const_cast, but I would rather see `set_ver` function accept `const char*`.

Reply:
I have made the set_ver function accept const char * by calling StripWord before this function. const_cast will be used for this purpose (outside the set_ver function).